model/user.go

```diff
@@ -0,0 +1,17 @@
+package model
+
+import (
+	"time"
+)
+
+type User struct {
```

Comment:
Exported structure has to have a comment.

Reply:
Added comment for structure User